src/odemis/gui/win/acquisition.py

```diff
@@ -610,6 +613,11 @@ def on_acquisition_done(self, future):
         # Make sure the file is not overridden
         self.btn_secom_acquire.Enable()
 
+# Step value for z stack levels
+ZSTEP = 1e-6  # m
+# Allowed zsteps range for zstack levels
+MINZ = 1
+MAXZ = 51
```

Comment:
It's already in the XRC file, so it should be fine, right? Let's drop the extra check.